pnpm-lock.yaml

```diff
@@ -1177,6 +1177,9 @@ importers:
     dependencies:
```

Comment:
It seems dependencies for visualping were not added to `pnpm-lock.yaml` file. Can you run `npx pnpm install`?